site/_includes/partials/navigation-rail.njk

```diff
@@ -3,10 +3,10 @@
 <navigation-rail role="navigation" class="lg:pad-left-200 lg:pad-right-200" aria-label="{{ 'i18n.common.primary' | i18n(locale) }}" tabindex="-1">
   <div class="display-flex lg:display-none align-center pad-200 hairline-bottom">
     <button class="navigation-rail__close button display-flex align-center justify-content-center width-700 height-700" aria-label="close">
-      {{ icon('close') }}
+      {{ icon('close', {label: 'close'}) }}
```

Comment:
Should this label be localized? The button already has an appropriate aria-label, should this icon be hidden from the accessibility tree?

Reply:
No, good call. I've hidden it.

Reply:
No, good call. I've hidden it.